card/card.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/card/card.css" />
+    <script
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Cards</h1>
+    <div class="card vertical-card txt-card box-shadow border-rad-md">
+      <h1>TITLE</h1>
+      <h2>Subtitle</h2>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit officia
+        totam iure, omnis dolorem reprehenderit cupiditate possimus provident
+        harum, nulla minima hic, ratione aspernatur odit esse assumenda iste
+        beatae asperiores perspiciatis ad impedit labore. Porro hic sed
+        perferendis magnam animi inventore?
+      </p>
+    </div>
+    <div
+      class="card vertical-card card-dismiss txt-card box-shadow border-rad-md"
+    >
+      <h1>TITLE</h1>
+      <i class="dismiss-icon fa-solid fa-xmark"></i>
+      <h2>Subtitle</h2>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit officia
+        totam iure, omnis dolorem reprehenderit cupiditate possimus provident
+        harum, nulla minima hic, ratione aspernatur odit esse assumenda iste
+        beatae asperiores perspiciatis ad impedit labore. Porro hic sed
+        perferendis magnam animi inventore?
+      </p>
+    </div>
+    <div class="card vertical-card box-shadow border-rad-md">
+      <img class="border-rad-md" src="/card/src/image.jpg" alt="demo image" />
```

Comment:
Maybe use more descriptive name for alt attribute for accessibility purpose.

Reply:
Thanks, sure will do this.

Reply:
Have updated the image description, please approve PR. Thanks.